cavern/build.gradle

```diff
@@ -53,12 +49,12 @@ dependencies {
     compile 'org.opencadc:cadc-vosi:[1.0,)'
     compile 'org.opencadc:cadc-rest:[1.0,)'
     compile 'org.opencadc:cadc-uws:[1.0,)'
-    compile 'org.opencadc:cadc-uws-server:[1.0.4,)'
+    compile 'org.opencadc:cadc-uws-server:[1.2.5,)'
     compile 'org.opencadc:cadc-cdp:[1.0,)'
     compile 'org.opencadc:cadc-gms:[1.0.0,)'
     compile 'org.opencadc:cadc-access-control:[1.1.1,2.0)'
     compile 'org.opencadc:cadc-access-control-identity:[1.0.3,2.0)'
-    compile 'org.opencadc:cadc-vos:[1.1.2,)'
+    compile 'org.opencadc:cadc-vos:[1.1.10,)'
```

Comment:
cadc-access-control-* should be declared as runtime dependencies so we don't code against any extended features... still not a dynamic plugin so cavern isn't really usable by others but that's a war/image build/deploy issue we have to work out at some  point.